handshake_messages.go

```diff
@@ -20,7 +20,7 @@ type clientHelloMsg struct {
 	serverName                   string
 	ocspStapling                 bool
 	scts                         bool
-	ems                          bool
+	ems                          bool // [UTLS]
```

Comment:
This comment could use some expansion.

Reply:
Throughout the standard files, which we are not supposed to change too much, I was leaving `[UTLS]` comment to explicitly state what we added.
I can switch to something like `[Added by UTLS]`, if that's better

Reply:
I understood that it was marked as added.  I was hoping you would expand the comment to explain _why_ it was added.

Reply:
Alright, `[UTLS]` -> `[UTLS] extension implemented due to its prevalence`